src/Reindex.php

```diff
@@ -73,9 +77,14 @@ public function run(): Response
         return $this->_lastResponse;
     }
 
+    /**
+     * @param bool $value
+     */
     public function setWaitForCompletion($value): void
     {
-        \is_bool($value) && $value = $value ? 'true' : 'false';
+        if (\is_bool($value)) {
+            $value = $value ? 'true' : 'false';
+        }
```

Comment:
```suggestion
        } else {
            \trigger_deprecation('ruflin/elastica', '7.1.6', 'Passing anything else than a boolean as 1st argument to "%s()" is deprecated, pass a boolean instead. It will be removed in 8.0.', __METHOD__);
        }
```
As @thePanz suggested a deprecation like this would allow to type-hint more strictly the parameter in next major version.

Reply:
Sure! thanks for the suggestion. It would be nice to be able to add missing parameter type declarations in PHPDoc, so for the next major release we can move them to the method definition.